.github/issue_template.md

```diff
@@ -0,0 +1,48 @@
+**Important:** Join our [slack channel](http://cilium.io/slack) for immediate
+support and an opportunity to ask questions. Please also have a look at our
+[FAQ](https://goo.gl/qG2YmU) for frequently asked questions first.
+
+If you open a GitHub issue, please follow these guidelines:
+
+1. It must be a bug or a feature request.
+2. The form below must be filled out.
+
+--------------
+
+## Bug reports
+
+### Title
+
+Choose a descriptive title that summarizes the symptom of the bug
+
+### General Information
+
+- Cilium version (run `cilium version`)
+- Kernel version (run `uname -a`)
+- Orchestration system version in use (e.g. Kubernetes 1.7, Mesos, own)
+- Link to relevant artifacts (policies, deployments scripts, ...)
```

Comment:
users: "What artifacts?" type an example of a command `kubectl get cnp -o json`

Reply:
Maybe we should defer requiring specific commands to be run until after have the bugtool (#1942) and then include what is left?

Reply:
That's true... looking through that list, should we also ask the users to paste the `ip address show` and `ip route show`?

Reply:
Yes, that's already on the list (`ip r` and `ip a`)

Reply:
I was asking if it should be on the issue template.

Reply:
I think we can use this for now. As people use this, we can update it to reflect what we think should be added over time.